app.js

```diff
@@ -9,18 +11,12 @@ app.route('/api/categories')
     .get(getCategories)
     .all(methodNotAllowedHandler(['GET']));
 
-app.all('/*', (req, res, next) => {
-    res.status(404).send({msg: 'route not found'})
-})
+app.route('/api/reviews/:review_id')
+    .get(getReview)
+    .all(methodNotAllowedHandler(['GET'])); //update with new method when adding
 
```

Comment:
I would defs keep the idea of the 405 there as but maybe leave the logic until a later ticket. There's a ticket in the extra stuffs about routing and this will help you add 405s to all your endpoints. And ... further DRY up your app